tekton/resources.yaml

```diff
@@ -0,0 +1,145 @@
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: tekton-pipelines-git
+spec:
+  type: git
+  params:
+  - name: url
+    value: https://github.com/tektoncd/pipeline # REPLACE with your own fork
+  - name: revision
+    value:  vX.Y.Z-invalid-tags-boouuhhh # REPLACE with your own commit
```

Comment:
NIT: what about having a fake sha1 as an example? Or are we saying we should create the git tag first?
I think that as long as we do testing before releasing the order should be:
- run this pipeline
- if successful, create a git tag

The alternative could be to create minor versions if test fails. Or we could change the pipeline to:
- run tests
- publish a git tag
- publish images

Even though hopefully in future we can fully automate this, and when we create a new tag run this pipeline - assuming github runs a webhook for that, but I I would assume it does.